src/ai/ai.cpp

```diff
@@ -4359,6 +4366,13 @@ void update_land_constructions(sys::state& state) {
 			}
 		}
 
+		// mobilize if we feel threatened
+		if(state.world.nation_get_ai_is_threatened(n)) {
+			military::start_mobilization(state, n);
```

Comment:
No. Mobilization really, really hurts your economy. Mobilizing when you don't immediately need the troops thus leaves you weaker and weaker over time. It also makes your pops more and more angry. *And* there are negative events that fire if you are mobilized while not at war.